src/perf_tool_backend.ml

```diff
@@ -500,22 +545,25 @@ module Perf_line = struct
     (module struct
       open Core
 
-      let check s = to_event s ~perf_map:None |> [%sexp_of: Event.t] |> print_s
+      let check s =
+        to_event s ~perf_map:None |> Option.value_exn |> [%sexp_of: Event.t] |> print_s
+      ;;
 
```

Comment:
We should add a couple of tests here showing `cbr`/`psb offs` parsing, so we don't accidentally regress it.